vars/runIntegTests.groovy

```diff
@@ -0,0 +1,27 @@
+void call(Map args = [:]) {
+    git url: 'https://github.com/opensearch-project/opensearch-build.git', branch: 'main'
+
+    def lib = library(identifier: 'jenkins@20211123', retriever: legacySCM(scm))
+    String manifest = args.manifest ?: "manifests/${INPUT_MANIFEST}"
+    def inputManifest = lib.jenkins.InputManifest.new(readYaml(file: manifest))
+
+    String tarball = args.architecture == "x64" ? env.ARTIFACT_URL_linux_x64 : env.ARTIFACT_URL_linux_arm64
+    String artifactRootUrl = args.architecture == "x64" ? env.ARTIFACT_ROOT_URL_linux_x64 : env.ARTIFACT_ROOT_URL_linux_arm64
+    String buildManifest = "${artifactRootUrl}/dist/opensearch/manifest.yml"
+    String testManifest = manifest.replaceAll(".yml", "-test.yml")    
```

Comment:
The reason I didn't was to prevent adding a param in `distribution-build.jenkinsfile`, and instead using the existing `INPUT_MANIFEST` to derive it. I'm ok either way, lmk if you have a preference.

Reply:
I'd rather be explicit with a parameter.  Note; if we had input manifests reference the test manifest this would also be a way to hold this information without complicating this interface

Reply:
Sure, added `TEST_MANIFEST` param in https://github.com/opensearch-project/opensearch-build/pull/1429/commits/15d3640d54eaddc6ec2b130cbf6575b50f2a7f32